example/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "fh-sync-cloud-example",
+  "version": "0.2.0",
+  "dependencies": {
+    "body-parser": "^1.0.2",
+    "cors": "~2.2.0",
+    "express": "~4.0.0",
+    "fh-mbaas-api": "7.0.0-1",
+    "fh-sync": "1.0.0",
+    "mongodb": "^2.1.18",
```

Comment:
I think this will fail on certain RHMAP instances. It needs to be `~2.1.18` since `2.2` has some issue with some URL formats.